src/index.js

```diff
@@ -4,7 +4,7 @@ import {
 } from './domElement.js';
 import './style.css';
 
-const taskstore = new TaskStore();
+var taskstore = new TaskStore();
```

Comment:
- [ ] Kindly use the let keyword when defining variables. Variables defined with var inside of a function “lose scope” and are essentially destroyed when the function exits. Follow the correct JS practices.